README.md

```diff
@@ -76,205 +62,10 @@ The generator will ask you a few questions about your new application and will g
-```
-
-The sub-generator will ask you for the module name under which you would like to create your new config, and will create a new AngularJS config file in that module's **config** folder.
-
-
-
-## AngularJS Test Sub-Generator
-
-Your MEAN application comes pre-bundled with the Karma test runner and Jasmine testing framework. To test your AngularJS controllers you’ll need to create a test file, which Karma will later use to run the tests. For this purpose we provided you with the AngularJS test sub-generator. Creating a new AngularJS test is effortless, just execute this command:
-
-
-```
-$ yo meanjs:angular-test <controller-name>
-```
-
-This will create a test file for your controller, and if the sub-generator doesn’t find the specified controller file, it will create one for you.
-
-
-**Don’t forget!** You're suppose to pass the controller name as an argument.
-
-
-
-## Express Model Sub-Generator
-
-Often you will find the need to just create a single Express model. The Express model sub-generator will help you with creating an Express model in the **app/models** folder. To create a new model just use *yo*:
-
-
-```
-$ yo meanjs:express-model <model-name>
-```
-
-This will create a new empty model in the **app/models** folder and a test file in the **app/tests** folder.
-
-
-**Note:** It is recommended you avoid naming your model in plural form and use a singular form instead. i.e article and not articles
-
-
-
-## Express Controller Sub-Generator
-
-Another recurring task is creating an empty Express controller. The Express controller sub-generator will help you with creating an Express controller in the **app/controllers** folder. To create a new controller just use *yo*:
-
-
-```
-$ yo meanjs:express-controller <controller-name>
-```
-
-This will create a new empty controller in the **app/controllers** folder.
-
-
-
-## Express Routes Sub-Generator
-
-To make HTTP requests to your controller methods you will need to use a routing file in the **app/routes** folder. The Express routes sub-generator will help you to add a new empty routes file. To create a new routes file execute this *yo* command:
-
-
-```
-$ yo meanjs:express-route <route-name>
-```
-
-This will create a new empty route file in the **app/routes** folder.
-
-
-## Express Test Sub-Generator
-
-Your MEAN application comes pre-bundled with the Mocha testing framework. To test your Express models you’ll need to create a new test file, which Mocha will use while running tests. For this purpose we provided you with the Express test sub-generator. Creating a new Express test is easy, just execute this command:
-
-
-```
-$ yo meanjs:express-test <model-name>
-```
-
-This will create a test file for your Express model, and if the sub-generator doesn’t find the specified model, it will create one for you.
-
-
-**Don’t forget!** You're supposed to pass the model name as an argument.
-
 ## License
 
 [BSD license](http://opensource.org/licenses/bsd-license.php)
```

Comment:
So you would like a capital .MD instead of lowercase .md? The files in the main repo are all .md, so I kept it like this for consistency.